### PR TITLE
Vice.js -- fix eslint warnings

### PR DESCRIPTION
**Pull request by retorquere**

### CODE REVIEW

**Comment by zuphilip** [comment]
Let me know when you updated this one.

**Comment by retorquere** [comment]
I've now updated it.

**Comment by retorquere** [comment]
I'm not sure why the linter fails? I don't see tabs in the files at those spots. Looking into it, but in the Travis logs, and locally, there are tabs there.

**Comment by retorquere** [comment]
Ah, new linter rules that I needed to install.

File: Vice.js [hunk 1]
```diff
@@ -38,12 +38,22 @@
 
 
 // attr()/text() v2 per https://github.com/zotero/translators/issues/1277
-function attr(docOrElem,selector,attr,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.getAttribute(attr):null;}function text(docOrElem,selector,index){var elem=index?docOrElem.querySelectorAll(selector).item(index):docOrElem.querySelector(selector);return elem?elem.textContent:null;}
+function attr(docOrElem, selector, attr, index) {
```
Comment (zuphilip) [inline Vice.js]: These lines can be deleted completely, as text/attr are now global functions from Zotero itself.

Comment (retorquere) [reply Vice.js]: I can add a linter rule for that if so desired. Would be an easy rule to do.

Comment (zuphilip) [reply Vice.js]: That would be great! Here is the corresponding issue #1913.

Comment (retorquere) [reply Vice.js]: https://github.com/zotero/translators/pull/1961

**Pull request merged**